### PR TITLE
dotCMS/core#17901 Show error message when try to delete and update r…

### PR DESCRIPTION
**Pull request by freddyucv**
- Show permission error when a limited user try to update or delete a rule

https://github.com/dotCMS/core-web/pull/1285/files#diff-2f81c9d19a8208425bfaa24e0f21564eR164

- Notify a error when a limited user try to add a new condition

https://github.com/dotCMS/core-web/pull/1285/files#diff-b59af840dd54464ba3cd28a8269597c3R181

- Notify a error when a limited user try to add a new group condition

https://github.com/dotCMS/core-web/pull/1285/files#diff-63d0117db0f2415fab7c86ee3bcb6b8bR184

- Recover right rule state after a permission error:

https://github.com/dotCMS/core-web/pull/1285/files#diff-2f81c9d19a8208425bfaa24e0f21564eR166

https://github.com/dotCMS/core-web/pull/1285/files#diff-2f81c9d19a8208425bfaa24e0f21564eR530

- Clean permission error message after a few second, if not then the message error is show forever

https://github.com/dotCMS/core-web/pull/1285/files#diff-2f81c9d19a8208425bfaa24e0f21564eR667

### CODE REVIEW

File: projects/dot-rules/src/lib/services/ConditionGroup.ts [hunk 1]
```diff
@@ -97,6 +103,7 @@ export class ConditionGroupService {
     }
 
     createConditionGroup(ruleId: string, model: ConditionGroupModel): Observable<any> {
+        this.loggerService.info('TESTING--------------');
```
Comment (hmoreras) [inline projects/dot-rules/src/lib/services/ConditionGroup.ts]: remove

File: projects/dot-rules/src/lib/services/ConditionGroup.ts [hunk 2]
```diff
@@ -157,17 +164,24 @@ export class ConditionGroupService {
 
     private _catchRequestError(operation): Func {
         return (err: any) => {
+            let message = null;
+
             if (err && err.status === HttpCode.NOT_FOUND) {
-                this.loggerService.info('Could not ' + operation + ' Condition: URL not valid.');
+                message = 'Could not ' + operation + ' Condition: URL not valid.';
```
Comment (hmoreras) [inline projects/dot-rules/src/lib/services/ConditionGroup.ts]: labels for this message ?

File: projects/dot-rules/src/lib/rule-engine.container.ts [hunk 3]
```diff
@@ -156,6 +158,18 @@ export class RuleEngineContainer {
       this.state.loading = false;
       this.state.showRules = false;
     });
+
+
+    merge(
+      this._ruleActionService.error,
+      this._conditionGroupService.error,
+      this._conditionService.error).subscribe((message: string) => {
```
Comment (hmoreras) [inline projects/dot-rules/src/lib/rule-engine.container.ts]: takeUntil ?

File: projects/dot-rules/src/lib/services/ConditionGroup.ts (hunk 2 above)
Comment (freddyucv) [reply projects/dot-rules/src/lib/services/ConditionGroup.ts]: really this a message to show in the console.log, and really this change is not necessary

File: projects/dot-rules/src/lib/services/ConditionGroup.ts (hunk 1 above)
Comment (freddyucv) [reply projects/dot-rules/src/lib/services/ConditionGroup.ts]: done https://github.com/dotCMS/core-web/pull/1285/commits/fcd832c15fcb74668f088a630f2024edb0f19fae#diff-63d0117db0f2415fab7c86ee3bcb6b8bL106

File: projects/dot-rules/src/lib/services/ConditionGroup.ts (hunk 2 above)
Comment (freddyucv) [reply projects/dot-rules/src/lib/services/ConditionGroup.ts]: done https://github.com/dotCMS/core-web/pull/1285/commits/fcd832c15fcb74668f088a630f2024edb0f19fae#diff-63d0117db0f2415fab7c86ee3bcb6b8bL167-R172

File: projects/dot-rules/src/lib/rule-engine.container.ts (hunk 3 above)
Comment (freddyucv) [reply projects/dot-rules/src/lib/rule-engine.container.ts]: done https://github.com/dotCMS/core-web/pull/1285/commits/fcd832c15fcb74668f088a630f2024edb0f19fae#diff-2f81c9d19a8208425bfaa24e0f21564eR168

File: projects/dot-rules/src/lib/services/Action.ts [hunk 4]
```diff
@@ -180,7 +186,8 @@ and should provide the info needed to make the user aware of the fix.`);
     private _catchRequestError(
         operation
     ): (response: Response, original: Observable<any>) => Observable<any> {
-        return (response: Response, original: Observable<any>): Observable<any> => {
+        return (response: Response): Observable<any> => {
+            console.log('response', response);
```
Comment (fmontes) [inline projects/dot-rules/src/lib/services/Action.ts]: remove.

Comment (freddyucv) [reply projects/dot-rules/src/lib/services/Action.ts]: done https://github.com/dotCMS/core-web/pull/1285/commits/d6b0f18039776c9caaad00a6474a085db4b1ae5b#diff-39ce5c57c2a64007ec9d26af36c3125fL190

**Pull request merged**